amplipi/streams.py

```diff
@@ -605,26 +597,23 @@ def disconnect(self):
     self.proc = None
     self.src = None
 
-  def info(self):
+  def info(self) -> models.SourceInfo:
     src_config_folder = f"{utils.get_folder('config')}/srcs/{self.src}"
     loc = f'{src_config_folder}/currentSong'
+    source = models.SourceInfo(img_url=self.logo)
     try:
       with open(loc, 'r') as file:
-        d = {}
         data = json.loads(file.read())
-
-        d['artist'] = data['artist']
-        d['song'] = data['song']
-        d['img_url'] = self.logo
-        d['station'] = data['station']
-
-        return d
+        source.artist = data['artist']
+        source.track = data['track']
+        source.station = data['station']
+        return source
```

Comment:
Does this block still work properly? I see artist/song/logo/station in the older code, with artist/TRACK/station in the new code. Does track need to be named song? Where did logo go?

Reply:
standardized this field on `track` which we were using everywhere else